test/mochaSetup.ts

```diff
@@ -0,0 +1,6 @@
+import * as chai from 'chai';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
we could specify `ignoreFiles` in ts.config (dunno which is better)

Reply:
:+1: